lib/Runtime/Language/DynamicProfileInfo.h

```diff
@@ -36,7 +37,7 @@ namespace IR
     BailOutKind EquivalentToMonoTypeCheckBailOutKind(BailOutKind kind);
 }
 
-#if ENABLE_DEBUG_CONFIG_OPTIONS
+#if ENABLE_DEBUG_CONFIG_OPTIONS || defined(REJIT_STATS)
```

Comment:
>REJIT_STATS [](start = 43, length = 11)

REJIT_STATS seems to be always defined when ENABLE_DEBUG_CONFIG_OPTIONS is defined